### ISSUE TITLE
Markin interprets decorators in fenced code-blocks as gherkin tags

### ISSUE THREAD

**Issue by migueldedios-wf** [fulstech/markin#9]
When entering the following text in a Markin-enabled Description field of a Jira ticket:

    ```python
    @decorator
    def function(x):
        pass
    ```

The ticket’s description is ultimately rendered as:

```
%gherkin-tag,@decorator%
def function(x):
    pass
```

For a demonstration, see the attached GIF.

![markin-gherkin-bug](https://cloud.githubusercontent.com/assets/1479351/15487852/2f9d1d24-2116-11e6-9923-c580880afa19.gif)

**Comment by davecasleton-wf** [comment]
It also interprets xml similarly: `` `<tag/>` `` becomes `%gherkin-variable,<tag>%%gherkin-variable,</tag>%`

**Comment by fulstech** [comment]
Thank you for your bug report. This issue has been fixed in Markin 3.0.0. You can get the latest version here https://marketplace.atlassian.com/plugins/com.fulstech.jira-gherkin-custom-field/server/overview.

Please don't hesitate to get back to us if you have any request. We always appreciate your suggestion.
